SEDaily-IOS/CustomTabViewController.swift

```diff
@@ -145,4 +154,38 @@ extension CustomTabViewController {
     func presentLeftSideMenu() {
         present(SideMenuManager.menuLeftNavigationController!, animated: true, completion: nil)
     }
+
+    fileprivate func askForReview() {
+        let popup = PopupDialog(title: "Hello! 👋",
+                                message: "Enjoying the SEDaily app?",
+                                gestureDismissal: false)
+        let feedbackPopup = PopupDialog(title: "Oh, sorry you're not liking the SEDaily app",
+                                        message: "Would you help us out by sending us your feedback?")
+        let feedbackYesButton = DefaultButton(title: "Sure! 👍") {
```

Comment:
Be careful of thumbs up gesture. It can mean different things in different countries.

Reply:
This is true. The text would most likely be localized anyway and we'd remove the thumbs up.